init.vim

```diff
@@ -2,47 +2,30 @@
 " === Plugins
 " ============================================================
 call plug#begin()
-Plug 'neoclide/coc.nvim' , {'branch': 'release'}
-Plug 'neoclide/coc-eslint', {'do': 'yarn install --frozen-lockfile'}
-Plug 'neoclide/coc-json', {'do': 'yarn install --frozen-lockfile'}
-Plug 'neoclide/coc-pairs', {'do': 'yarn install --frozen-lockfile'}
-Plug 'neoclide/coc-prettier', {'do': 'yarn install --frozen-lockfile'}
-Plug 'neoclide/coc-snippets', {'do': 'yarn install --frozen-lockfile'}
-Plug 'neoclide/coc-tsserver', {'do': 'yarn install --frozen-lockfile'}
-Plug 'HerringtonDarkholme/yats.vim'
-Plug 'maxmellon/vim-jsx-pretty'
-Plug 'Shougo/denite.nvim'
-Plug 'morhetz/gruvbox'
 Plug 'tpope/vim-fugitive'
-""Plug 'tpope/vim-surround'
-""Plug 'tpope/vim-speeddating'
-""Plug 'tpope/vim-sleuth'
 Plug 'tpope/vim-unimpaired'
 Plug 'tpope/vim-vinegar'
-"Plug 'junegunn/gv.vim'
-"Plug 'mmai/vim-markdown-wiki'
```

Comment:
I may want to reconsider having removed some of these. Although I think there are possibly more modern alternatives which I should consider too.

Reply:
Techdebt ticket raised https://github.com/js-jslog/dotfiles/pull/7#discussion_r607311492